tests/helpers/test_service.py

```diff
@@ -251,9 +263,11 @@ def test_fail_silently_if_no_service(self, mock_log):
 
     call = ha.ServiceCall("light", "turn_on", {"area_id": "test-area"})
 
-    assert {"light.in_area"} == await service.async_extract_entity_ids(hass, call)
+    assert {"light.in_area"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4603109)